lib/BUILD

```diff
@@ -0,0 +1,6 @@
+cc_library(
+    name = "hello_world_lib",
```

Comment:
I think we should name it something like scsi_to_nvme_lib.

Reply:
Yup that's a good point. I'll make the change

Reply:
Actually, on second thoughts we can just leave it as it is and remove this whole block altogether as we add more code. In Bazel, we can define multiple libraries inside a single build file. So, in this case I defined a `hello_world_lib` and when we start adding the actual code for the translation, we can create multiple libraries so that it's easier to individually test them. This also gives us freedom to include individual libraries in case there's a dependency. For example, we can define something like

```
cc_library(
       name = "scsi_to_nvme_thread_manager",
       srcs = ["thread_manager.cc"],
       ....
)

cc_library(
       name = "scsi_to_nvme_standard_output",
       srcs = ["standard_output.cc"],
       ....
)
```
This way we can individually import only the `thread manager` and the `standard output` into other packages without importing anything else from this package. I can add a comment above this to make sure we remove it once we add the actual translation code.

Thoughts on this?

Reply:
That makes sense to me. I like the idea that we can remove all the `hello_world` code/blocks together once we start adding actual code. Adding a `TODO` comment would be helpful!